modules/nw-ingress-sharding.adoc

```diff
@@ -6,25 +6,35 @@
 [id="nw-ingress-sharding_{context}"]
 = Ingress Controller sharding
 
-You can use Ingress sharding, also known as router sharding, to distribute a set of routes across multiple routers by adding labels to routes, namespaces, or both. The Ingress Controller uses a corresponding set of selectors to admit only the routes that have a specified label. Each Ingress shard comprises the routes that are filtered using a given selection expression.
+You can use Ingress sharding, also known as router sharding, to distribute a set of routes across multiple routers by adding labels to routes, namespaces, or both. The Ingress Controller uses a corresponding set of selectors to admit only the routes that have a specified label. Each Ingress shard comprises the routes that are filtered by using a given selection expression.
 
 As the primary mechanism for traffic to enter the cluster, the demands on the Ingress Controller can be significant. As a cluster administrator, you can shard the routes to:
 
-* Balance Ingress Controllers, or routers, with several routes to speed up responses to changes.
-* Allocate certain routes to have different reliability guarantees than other routes.
+* Balance Ingress Controllers, or routers, with several routes to accelerate responses to changes.
+* Assign certain routes to have different reliability guarantees than other routes.
 * Allow certain Ingress Controllers to have different policies defined.
 * Allow only specific routes to use additional features.
 * Expose different routes on different addresses so that internal and external users can see different routes, for example.
-* Transfer traffic from one version of an application to another during a blue green deployment.
+* Transfer traffic from one version of an application to another during a blue-green deployment.
 
-When Ingress Controllers are sharded, a given route is admitted to zero or more Ingress Controllers in the group. A route's status describes whether an Ingress Controller has admitted it or not. An Ingress Controller will only admit a route if it is unique to its shard.
+When Ingress Controllers are sharded, a given route is admitted to zero or more Ingress Controllers in the group. The status of a route describes if an Ingress Controller has admitted the route or not. An Ingress Controller only admits a route if the route is unique to a shard.
 
-An Ingress Controller can use three sharding methods:
+With sharding, you can distribute subsets of routes over multiple Ingress Controllers. These subsets can be non-overlapping, also called _traditional_ sharding, or overlapping, otherwise known as _overlapped_ sharding.
```

Comment:
per ISG
```suggestion
With sharding, you can distribute subsets of routes over multiple Ingress Controllers. These subsets can be nonoverlapping, also called _traditional_ sharding, or overlapping, otherwise known as _overlapped_ sharding.
```